common/taskana-common-test/src/main/resources/taskana-test.ldif

```diff
@@ -332,6 +337,22 @@ ou: Organisationseinheit/Organisationseinheit B
 cn: Brunhilde Bio
 userPassword: user-b-2
 
+########################
+# Users in other cn
+########################
+dn: uid=otheruser,cn=other-users,OU=Test,O=TASKANA
+objectclass: inetorgperson
+objectclass: organizationalperson
+objectclass: person
+objectclass: top
+givenName: Other
+description: User in other cn than search root
+uid: otheruser
+sn: User
+ou: Other
+cn: Other User
+userPassword: otheruser
+
```

Comment:
Please also add these changes to `taskana-example.ldif`

Reply:
This is just a test data configuration. I don't think that we need this one in the other file. Do you disagree?

Reply:
I prefer to have those two filed completely identical, so that we can reduce them to one file some day in the future